From a3feeb63145506e3d77be99e1dd214247ebabe1a Mon Sep 17 00:00:00 2001
From: Stefan Berger <stefanb@linux.vnet.ibm.com>
Date: Sun, 28 Feb 2021 16:35:56 -0500
Subject: [PATCH 03/12] Set CRYPTOLIBRARY to openssl

---
 utils/reg.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/reg.sh b/utils/reg.sh
index 24ea1df..c9d49b9 100755
--- a/utils/reg.sh
+++ b/utils/reg.sh
@@ -281,7 +281,7 @@ export WARN
 export PREFIX
 export -f initprimary
 # hack because the mbedtls port is incomplete
-export CRYPTOLIBRARY=`${PREFIX}getcryptolibrary`
+export CRYPTOLIBRARY=openssl
 
 # example for running scripts with encrypted sessions, see TPM_SESSION_ENCKEY=getrandom below
 export TPM_SESSION_ENCKEY
-- 
2.39.1

