From 223a78820b10fcce8b65d5827d1699dab49e45e1 Mon Sep 17 00:00:00 2001
From: Stefan Berger <stefanb@linux.vnet.ibm.com>
Date: Sun, 28 Feb 2021 16:42:11 -0500
Subject: [PATCH 4/9] Store volatile state at every step

---
 utils/reg.sh | 14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

diff --git a/utils/reg.sh b/utils/reg.sh
index 33e3299..de7e181 100755
--- a/utils/reg.sh
+++ b/utils/reg.sh
@@ -124,6 +124,13 @@ printUsage ()
     echo "-51 Events"
 }
 
+storeVolatileState()
+{
+    echo ">>> Storing volatile state <<<"
+    ${SWTPM_IOCTL} -v --tcp ${TPM_SERVER_NAME}:${TPM_PLATFORM_PORT}
+}
+export -f storeVolatileState
+
 checkSuccess()
 {
 if [ $1 -ne 0 ]; then
@@ -133,7 +140,7 @@ if [ $1 -ne 0 ]; then
 else
     echo " INFO:"
 fi
-
+storeVolatileState
 }
 
 # FIXME should not increment past 254
@@ -146,6 +153,7 @@ if [ $1 -ne 0 ]; then
 else
     echo " INFO:"
 fi
+storeVolatileState
 }
 
 checkFailure()
@@ -157,6 +165,7 @@ if [ $1 -eq 0 ]; then
 else
     echo " INFO:"
 fi
+storeVolatileState
 }
 
 cleanup()
@@ -252,6 +261,9 @@ initprimary()
 
 powerup()
 {
+    ${SWTPM_IOCTL} -i --tcp ${TPM_SERVER_NAME}:${TPM_PLATFORM_PORT}
+    # Do it a 2nd time now that the previously store volatile state is gone
+    # Now startup must be sent to the TPM again
     ${SWTPM_IOCTL} -i --tcp ${TPM_SERVER_NAME}:${TPM_PLATFORM_PORT}
     return $?
 }
-- 
2.26.2

