From 38bf1de5fca4bc1f63bfc49dcd9d0ff2155db8e1 Mon Sep 17 00:00:00 2001
From: Stefan Berger <stefanb@linux.vnet.ibm.com>
Date: Sun, 28 Feb 2021 16:33:02 -0500
Subject: [PATCH 5/5] Disable tests related to 'events'

---
 utils/regtests/testevent.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/regtests/testevent.sh b/utils/regtests/testevent.sh
index 24330bf..d72c680 100755
--- a/utils/regtests/testevent.sh
+++ b/utils/regtests/testevent.sh
@@ -1,5 +1,6 @@
 #!/bin/bash
 #
+exit 0
 
 #################################################################################
 #										#
-- 
2.47.0

