From 85a7cdc22a6a7d8ad7788e0b074ab05071521cae Mon Sep 17 00:00:00 2001
From: Stefan Berger <stefanb@linux.vnet.ibm.com>
Date: Sun, 28 Feb 2021 16:36:53 -0500
Subject: [PATCH 07/13] Disable rev155 test cases

---
 utils/regtests/testattest155.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/regtests/testattest155.sh b/utils/regtests/testattest155.sh
index 824bab7..0187652 100755
--- a/utils/regtests/testattest155.sh
+++ b/utils/regtests/testattest155.sh
@@ -1,5 +1,6 @@
 #!/bin/bash
 #
+exit 0
 
 #################################################################################
 #										#
-- 
2.39.1

