From 7a6fe0d006ef5f2c6c960aa324b3e0ca4e0afcc9 Mon Sep 17 00:00:00 2001
From: Stefan Berger <stefanb@linux.vnet.ibm.com>
Date: Sun, 28 Feb 2021 16:36:53 -0500
Subject: [PATCH 7/9] Disable rev155 test cases

---
 utils/regtests/testattest155.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/regtests/testattest155.sh b/utils/regtests/testattest155.sh
index 0bf88aa..554a40e 100755
--- a/utils/regtests/testattest155.sh
+++ b/utils/regtests/testattest155.sh
@@ -1,5 +1,6 @@
 #!/bin/bash
 #
+exit 0
 
 #################################################################################
 #										#
-- 
2.26.2

