From 737f062890fff5b97e0e704da76886738339422c Mon Sep 17 00:00:00 2001
From: Stefan Berger <stefanb@linux.vnet.ibm.com>
Date: Sun, 28 Feb 2021 16:37:27 -0500
Subject: [PATCH 08/13] Disable x509 test cases

---
 utils/regtests/testx509.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/regtests/testx509.sh b/utils/regtests/testx509.sh
index 17d0288..03650fe 100755
--- a/utils/regtests/testx509.sh
+++ b/utils/regtests/testx509.sh
@@ -1,5 +1,6 @@
 #!/bin/bash
 #
+exit 0
 
 #################################################################################
 #										#
-- 
2.39.1

