From bcfe5f203d7a5c1e298ea5674268b3174fa67f31 Mon Sep 17 00:00:00 2001
From: Stefan Berger <stefanb@linux.vnet.ibm.com>
Date: Sun, 28 Feb 2021 16:38:10 -0500
Subject: [PATCH 09/13] Disable getcapability TPM_CAP_ACT

---
 utils/regtests/testgetcap.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/utils/regtests/testgetcap.sh b/utils/regtests/testgetcap.sh
index f88697b..f5f9729 100755
--- a/utils/regtests/testgetcap.sh
+++ b/utils/regtests/testgetcap.sh
@@ -120,8 +120,8 @@ echo "Get Capability TPM_CAP_AUTH_POLICIES"
 ${PREFIX}getcapability -cap 9 -pr 40000000 -v > run.out
 checkSuccess $?
 
-echo "Get Capability TPM_CAP_ACT"
-${PREFIX}getcapability -cap a -pr 40000110 -v > run.out
-checkSuccess $?
+#echo "Get Capability TPM_CAP_ACT"
+#${PREFIX}getcapability -cap a -pr 40000110 -v > run.out
+#checkSuccess $?
 
 
-- 
2.39.1

