From 7c28def4e96e7340e848605da8130c16636dcc97 Mon Sep 17 00:00:00 2001
From: Stefan Berger <stefanb@linux.vnet.ibm.com>
Date: Tue, 6 May 2025 18:17:37 -0400
Subject: [PATCH] Remove unnecessary include of openssl/engine.h

---
 utils/tsscrypto.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/utils/tsscrypto.c b/utils/tsscrypto.c
index f2a2e3d..79cca8e 100644
--- a/utils/tsscrypto.c
+++ b/utils/tsscrypto.c
@@ -57,7 +57,6 @@
 #include <openssl/rsa.h>
 #endif
 #include <openssl/rand.h>
-#include <openssl/engine.h>
 #if OPENSSL_VERSION_NUMBER >= 0x30000000
 #include <openssl/core_names.h>
 #include <openssl/param_build.h>
-- 
2.49.0

