From 2fdd0e84a3ac04cc09e9630b3b825b66ff288340 Mon Sep 17 00:00:00 2001
From: Stefan Berger <stefanb@linux.ibm.com>
Date: Mon, 15 May 2023 10:32:49 -0400
Subject: [PATCH 13/13] Disable x509 test cases (part2)

---
 utils/regtests/testhelp.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/utils/regtests/testhelp.sh b/utils/regtests/testhelp.sh
index cc8ad56..33f8ad0 100755
--- a/utils/regtests/testhelp.sh
+++ b/utils/regtests/testhelp.sh
@@ -6972,6 +6972,9 @@ do
     # skip all EC Encrypt/Decrypt commands
     [[ ${CC} =~ ^(16c|190|199|19A)$ ]] && continue
 
+    # disable CC_CertifyX509 related tests
+    [[ ${CC} =~ ^(197|19A)$ ]] && continue
+
     echo "startauthsession"
     ${PREFIX}startauthsession -se p > run.out
     checkSuccess $?
-- 
2.39.1

